source/code/plugins/changetracking_lib.rb

```diff
@@ -12,6 +12,8 @@ def self.log= (value)
         @@log = value
     end
 
+    @@force_send_last_upload = Time.now
```

Comment:
This is just an internal variable. The time zone has no effect.
I'd rather have no state files but I see your concern. It is addressed since we send the data on each restart.